dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/jaxb/JAXBContextCache.java

```
@@ -0,0 +1,504 @@
+            } catch (ClassNotFoundException e) {
+                // do nothing
+            }
+
+        }
+        return added;
+    }
+
+    public static void addPackage(Set<Class<?>>  classes, String pkg, ClassLoader loader) {
+
+        boolean isLoggableInfo = LOG.isLoggable(Level.INFO);   // Liberty Change
+
+        try {
+            classes.add(Class.forName(pkg + ".ObjectFactory", false, loader));
+        } catch (Exception ex) {
+            //ignore
+	    // Liberty Change begin
+	    if (isLoggableInfo) {
+	       LOG.info("addPackage: Ignoring exception: " + ex);
+            } 
+	    // Liberty Change end
+        }
+        try (InputStream ins = loader.getResourceAsStream('/' + pkg.replace('.', '/') + "/jaxb.index");
+            BufferedReader reader = new BufferedReader(new InputStreamReader(ins, StandardCharsets.UTF_8))) {
+            if (!StringUtils.isEmpty(pkg)) {
+                pkg += '.';
+            }
+
+            String line = reader.readLine();
+            while (line != null) {
+                line = line.trim();
+                if (line.indexOf('#') != -1) {
+                    line = line.substring(0, line.indexOf('#'));
+                }
+                if (!StringUtils.isEmpty(line)) {
+                    try {
+                        Class<?> ncls = Class.forName(pkg + line, false, loader);
+                        classes.add(ncls);
+                    } catch (Exception e) {
+                        // ignore
+	                // Liberty Change begin
+	                if (isLoggableInfo) {
+	                   LOG.info("addPackage: Ignoring exception 1: " + e);
+                        } 
+		        // Liberty Change end
+                    }
+                }
+                line = reader.readLine();
+            }
+
+        } catch (Exception ex) {
+            //ignore
+	    // Liberty Change begin
+	    if (isLoggableInfo) {
+	       LOG.info("addPackage: Ignoring exception 2: " + ex);
```

Since this exception is ignored, this should be logged only in the trace.log. Check and log needs to change to fine